src/main/scala/com/databricks/spark/csv/CsvRelation.scala

```diff
@@ -136,6 +136,56 @@ case class CsvRelation protected[spark] (
     }
   }
 
+  override def buildScan(requiredColumns: Array[String]): RDD[Row] = {
```

Comment:
Right! I will add comments like the ones in `org.apache.spark.sql.sources.PrunedScan` at Spark
Also, let me update them (below) soon.